include/battle/battle.h

```diff
@@ -178,13 +179,14 @@ typedef struct UnkBattlemonSub {
     u32 magnetRiseTurns     : 3;
     u32 healBlockTurns      : 3;
     u32 embargoFlag         : 3;
-    u32 knockOffFlag        : 1; // unclear whether true mean knocked off or not knocked off based on current information on its usage
-    u32 metronomeTurns      : 4; // refers to the item, not the move
-    u32 micleBerryFlag      : 1;
-    u32 custapBerryFlag     : 1;
-    u32 quickClawFlag       : 1;
-    u32 meFirstFlag         : 1;
-    u32 unk4_2F             : 1; // unused
+    u32 knockOffFlag        : 1; // unclear whether true mean knocked off or not knocked
+                                 // off based on current information on its usage
+    u32 metronomeTurns  : 4;     // refers to the item, not the move
+    u32 micleBerryFlag  : 1;
+    u32 custapBerryFlag : 1;
+    u32 quickClawFlag   : 1;
+    u32 meFirstFlag     : 1;
+    u32 unk4_2F         : 1; // unused
```

Comment:
somehow splitting this comment across multiple lines broke bitfield alignment

Reply:
clang might just check sequential bitfield alignment tbh